### PR TITLE
Add "editor/posts#preview" to show styled preview for editor

### LINKED ISSUE #229: 記事の入稿者として、スタイルがあたった状態でのプレビューが見たい

### PR DESCRIPTION
**Pull request by myokoym**
Close #229

### CODE REVIEW

**Comment by myokoym** [comment]
Discussed here: https://github.com/bm-sms/daimon-news-multi-tenant/commit/38bc2d34f95816e86eec278dd335cc9522a2cffa#commitcomment-16108256

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -22,12 +22,12 @@
       .wrappable
         .wrappable__content.wrappable__content--flex.main-pane
           %main
-            = render 'breadcrumbs'
+            = render 'application/breadcrumbs'
```
Comment (tricknotes) [inline app/views/layouts/application.html.haml:25]: Is `application` needed?

File: app/views/editor/posts/index.html.haml [hunk 2]
```diff
@@ -21,6 +21,7 @@
         %td= post.author&.name
         %td= image_tag post.thumbnail_url, height: 80 if post.thumbnail.present?
         %td= link_to 'Show', editor_post_url(post)
+        %td= link_to 'Preview', preview_editor_post_url(post)
```
Comment (tricknotes) [inline app/views/editor/posts/index.html.haml]: An editor can't be back from preview page to editor page. So it is kindness that `target: '_blank' is specified.

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (tricknotes) [inline app/views/layouts/application.html.haml:25]: Ah, right. https://github.com/bm-sms/daimon-news-multi-tenant/commit/9e7702c64791b235a096259f3f2db5ce6fbdd866 explains all reasons.

Comment (myokoym) [inline app/views/layouts/application.html.haml:25]: Yes, an error of `ActionView::MissingTemplate` occurred if don't add `application`:

```
ActionView::Template::Error (Missing partial editor/posts/_breadcrumbs, editor/application/_breadcrumbs with {:locale=>[:ja], :formats=>[:html], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :haml]}. Searched in:
  * "/home/myokoym/work/sms/daimon-news-multi-tenant/app/views"
  * "/home/myokoym/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/kaminari-0.16.3/app/views"
  * "/home/myokoym/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/devise-3.5.6/app/views"
):
    22:       .wrappable
    23:         .wrappable__content.wrappable__content--flex.main-pane
    24:           %main
    25:             = render 'breadcrumbs'
    26:             = yield
    27:         %aside.wrappable__content.promotions
    28:           - if current_site.ad_client.present? && current_site.ad_slot.present?
  app/views/layouts/application.html.haml:25:in `_app_views_layouts_application_html_haml__3700888417442111477_69938080105520'
  app/controllers/editor/posts_controller.rb:56:in `preview'
```

File: app/views/editor/posts/index.html.haml (hunk 2 above)
Comment (myokoym) [inline app/views/editor/posts/index.html.haml]: Nice idea :star:

**Comment by tricknotes** [comment]
It seems almost good!
Additionally, it would be better if test for preview page is there!
It might be a very simple.

**Comment by myokoym** [comment]
Thanks! I'm going to add test.

**Comment by tricknotes** [comment]
:rocket:

**Pull request merged**